.circleci/config.yml

```diff
@@ -389,6 +392,24 @@ workflows:
             tags:
               only: /.*/
 
+      - python_job:
+          name: mypy type check
+          command: mypy --no-incremental --junit-xml job-results/mypy.xml .
+          has_results: true
+          allow_fail: true
```

Comment:
Wouldn't we want this one to be blocking?

Reply:
Yes, eventually. I want to give developers a week or so (or maybe during an upcoming work week) to get the tools installed locally first.